src/query_planner/abstract_table_plan.py

```diff
@@ -0,0 +1,36 @@
+"""Abstract class for all the scan planners
```

Comment:
What is a table plan?

Reply:
We thought we would need a table plan for the videos (the table representing the video).

Reply:
Have a look at the table_ref in src/query_parser/table_ref.py.

Reply:
Does this mean we should scrap the video_table_plan and abstract_table_plan and replace them with the table_ref?

Reply:
I have changed the references of video_table_plan and abstract_table_plan into table_ref.